central/docs/api_custom_routes/README.md

```diff
@@ -1,43 +1,22 @@
 # Custom Route API specs
-This directory includes manually created OpenAPI/Swagger specs for Central's custom routes defined in [/central/main.go](/central/main.go) (refer to `customRoutes()`)
-
-These particular specs are currently NOT accessible via Central, instead they are/will be available in the official docs.
-
-TODO(ROX-28173): improve doc creation, automation, serving, etc..
-
-## Manually Modifying and Converting to AsciiDoc
-
-_Disclaimer: Before executing these steps confirm with the docs team if the process, scripts, etc. is still accurate. These steps represent an initial PoC and can/should be improved, which is in scope for ROX-28173_
-
-The OpenAPI specs must be converted to AsciiDoc for publishing to the official docs.
 
-Sample steps to modify, validate, and convert these specs:
-
-1. Load the `yaml` into https://editor.swagger.io/ (or use the equiv offline variation) and make appropriate changes and validate syntax.
-   - _Refer to the [OpenAPI guide](https://swagger.io/docs/specification/v3_0/about/) and/or [official specs](https://spec.openapis.org/) for more details_
-
-2. Copy the modified `yaml` back into this repo
-
-3. Generate an AsciiDoc from the modified `yaml` (change `swagger_dir` and `srcfile` as appropriate)
-          
-       swagger_dir=/path-to-stackrox-repo/central/docs/api_custom_routes
-
-       srcfile=image_service_swagger.yaml
-
-       docker run --rm -v "$swagger_dir:/local" openapitools/openapi-generator-cli generate -i /local/$srcfile -g asciidoc -o /local/asciidoc
+This directory includes manually created OpenAPI/Swagger specs for Central's custom routes defined in [/central/main.go](/central/main.go) (refer to `customRoutes()`)
 
-4. Clone `rhacs-api-docs-gen` repo: https://github.com/gaurav-nelson/rhacs-api-docs-gen
+These specs will roll up under the v1 API inside of Central as well as in the official docs on docs.redhat.com.
 
-5. Move the generated `adoc` file to `scripts/` dir of `rhacs-api-docs-gen` tool
+## Creating a new spec for a custom route
 
-       mv $swagger_dir/asciidoc/index.adoc /path-to-rhacs-api-docs-gen-repo/scripts
+Creating a new spec for a custom route involves a few steps.
 
-6. Execute `scripts/updateasciidoc.js` to cleanup the `adoc` file (will be modified in place)
+If you want to maintain the spec in YAML:
 
-       cd /path-to-rhacs-api-docs-gen-repo
+1. Create a YAML-based swagger spec from scratch in the file format `<service_name>_swagger.yaml` in this directory.
```

Comment:
nitpick: Is there a reason for different file prefix naming convention? `service_name` for YAML vs `serviceName` for JSON?
tbh. I would prefer having the same name. Snake or camel, it's not relevant. Just to have nice sorting of files one after another.

Reply:
Yeah what was in there was snake case, but the existing swagger docs are camel case.  So I'll switch it all to came case.